wallet/addresses.go

```diff
@@ -758,6 +758,15 @@ func (w *Wallet) NewInternalAddress(ctx context.Context, account uint32, callOpt
 
 func (w *Wallet) newChangeAddress(ctx context.Context, op errors.Op, persist persistReturnedChildFunc,
 	accountName string, account uint32, gap gapPolicy) (stdaddr.Address, error) {
+	// Imported voting accounts must not be used for change.
+	//
+	// TODO: For now, all accounts above the imported account are voting
+	// accounts. This may change in the future. Check the account type
+	// rather than using the number.
```

Comment:
This does worry me a bit, as we already use the numeric account numbers over the imported account number to signify that an account is an imported xpub account.  Both those and these new "voting accounts" are of course BIP0044 but private key handling will be different between both, and I don't think we should be breaking deriving change addresses for imported xpubs either.

Reply:
Will work on solving the TODO.

Reply:
Hopefully this is resolved. Added type to AccountProperties.